config/application.rb

```diff
@@ -17,11 +17,6 @@ class Application < Rails::Application
     # configuring Nginx on Elastic Beanstalk is a pain.
     config.middleware.use Rack::Deflater
 
-    # The compile method (default in tinymce-rails 4.5.2) doesn't work when also
-    # using tinymce-rails-imageupload, so revert to the :copy method
-    # https://github.com/spohlenz/tinymce-rails/issues/183
```

Comment:
@mjgiarlo tinymce-rails-imageupload is still in the hyku gemfile. Maybe it should be removed?